typespec-tests/tsp/response.tsp

```diff
@@ -92,4 +106,22 @@ interface ResponseOp {
     @header("operation-location")
     operationLocation: ResourceLocation<Resource>;
   };
+
+  @route("/exists")
+  @head
+  exists is RpcOperationWithAdditionalResponse<
+    {},
+    {
+      @doc("A response containing headers related to the Job Schedule, if it exists.")
+      @statusCode
+      code: "200";
+    },
+    {
+      @doc("A response containing headers related to the Job Schedule, if it exists.")
+      @statusCode
+      code: "404";
```

Comment:
what does default status code 404 mean?

Reply:
that 404 is expected response (not an error)

swagger https://github.com/Azure/azure-rest-api-specs/blob/main/specification/batch/data-plane/Microsoft.Batch/stable/2022-10-01.16.0/BatchService.json#L2829-L2855

Reply:
ok, I see, thanks.